docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst

```diff
@@ -297,28 +321,54 @@ State and Transition Data Tables
      - TBL cancer sequelae prevalence
      - Not forecasted
    * - MST
-     - 2.53 (95% CI: 1.5, 3.88) years; YYY distribution of uncertainty
+     - 2.06 years (95% CI: 0.42 - 3.83); normal distrbution of uncertainty at draw level
```

Comment:
Engineers will appreciate if we convert this into the parameters for a specific distribution, such as a truncated normal.